lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -316,12 +336,14 @@ const cli = yargs
     'expectations-path': 'The path to the expected audit results file',
     'debug': 'Save the artifacts along with the output',
   })
-  .require('config-path')
-  .require('expectations-path')
+  .require('config-path', true)
+  .require('expectations-path', true)
   .argv;
 
 const configPath = resolveLocalOrCwd(cli['config-path']);
+/** @type {ExpectedLHR[]} */
 const expectations = require(resolveLocalOrCwd(cli['expectations-path']));
+// TODO: also add "ExpectedLHR" to each expectations module export?
```

Comment:
what is a checked static path? is that somehow declaring that `require(SmoketestDfn.expectations)` should be of type `ExpectedLHR`?

Reply:
well my idea was we could change `SmoketestDfn.expectations` from `'path/to/expectations.js'` to `require('path/to/expectations.js')`, but it passes the path into `smokehouse.js`, not the contents of the file, so that won't work anyways :)